### PR TITLE
Updated versions on install_from_source manual for Foreman 3.4

### PR DESCRIPTION
**Pull request by nofaralfasi**

### CODE REVIEW

File: _includes/manuals/nightly/3.4_install_from_source.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.5 or newer
-* NodeJS 10 or newer
-* NPM 3.0 or newer
+* Ruby 2.7 or newer
```
Comment (evgeni) [inline _includes/manuals/nightly/3.4_install_from_source.md]: Or newer is not exactly correct here, we currently do not support Ruby 3.0+

File: _includes/manuals/nightly/3.4_install_from_source.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
...
+* NodeJS 12 or newer
```
Comment (evgeni) [inline _includes/manuals/nightly/3.4_install_from_source.md]: Or newer is not exactly correct here, we currently do not support Node 16+.

**Review by evgeni — CHANGES_REQUESTED** [review]

File: _includes/manuals/nightly/3.4_install_from_source.md (hunk 2 above)
Comment (nofaralfasi) [reply _includes/manuals/nightly/3.4_install_from_source.md]: Thank you @evgeni, I will fix it.

**Comment by nofaralfasi** [comment]
`NodeJS 12 or 14` - is OK?

**Comment by evgeni** [comment]
I think so, yeah. @MariaAga has been cleaning this up in core recently.

**Comment by MariaAga** [comment]
for now node 12 is ok but its not really supported (https://endoflife.date/nodejs) so I would only write `Node 14`.

File: _includes/manuals/nightly/3.4_install_from_source.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.5 or newer
-* NodeJS 10 or newer
-* NPM 3.0 or newer
+* Ruby 2.7.x
+* NodeJS 12 or 14
+* NPM 6.0 or newer
```
Comment (MariaAga) [inline _includes/manuals/nightly/3.4_install_from_source.md]: We dont support npm 7+ (reasons described here: https://community.theforeman.org/t/javascript-infrastructure-update-plan/28614)

Comment (nofaralfasi) [reply _includes/manuals/nightly/3.4_install_from_source.md]: Thanks, so I will change it to `NPM 6.x`.

**Review by evgeni — APPROVED** [review]

**Review by Ron-Lavi — APPROVED** [review]
Thanks @nofaralfasi 👍🏼

**Pull request merged**